app/scripts/metamask-controller.js

```diff
@@ -1144,6 +1144,9 @@ export default class MetamaskController extends EventEmitter {
           `${this.approvalController.name}:addRequest`,
           `${this.approvalController.name}:acceptRequest`,
           `${this.approvalController.name}:rejectRequest`,
+          `${this.approvalController.name}:startFlow`,
+          `${this.approvalController.name}:showSuccess`,
+          `${this.approvalController.name}:showError`,
```

Comment:
It looks like we forgot to remove `acceptRequest` and `rejectRequest`, which are no longer messages that `TransactionController` sends.

Having said that. Shouldn't we wait to add these methods to the `TransactionController` messenger until we add the feature there? Otherwise we risk adding methods we might not need.

Reply:
100% I think it makes sense to not include these methods until we add the feature as you mentioned. Removed and pushed changes.